### PR TITLE
Update alerts fetching for plugin status cards

### PR DESCRIPTION
**Pull request by rawagner**
followup for https://github.com/openshift/console/pull/4539
/assign @dtaylor113

### CODE REVIEW

File: frontend/public/actions/dashboards.ts [hunk 1]
```diff
@@ -138,7 +131,6 @@ export type WatchPrometheusQueryAction = (
   namespace?: string,
   timespan?: number,
 ) => ThunkAction;
-export type WatchAlertsAction = () => ThunkAction;
 export type StopWatchURLAction = (url: string) => void;
 export type StopWatchPrometheusAction = (query: string, timespan?: number) => void;
 export type StopWatchAlertsAction = () => void;
```
Comment (dtaylor113) [inline frontend/public/actions/dashboards.ts]: Should this be removed like `WatchAlertsAction` has been?

File: frontend/packages/metal3-plugin/src/components/baremetal-hosts/dashboard/StatusCard.tsx [hunk 2]
```diff
@@ -66,7 +64,11 @@ const getHostHardwareHealthState = (obj): HostHealthState => {
 const filterAlerts = (alerts: Alert[]): Alert[] =>
   alerts.filter((alert) => _.get(alert, 'labels.hwalert'));
 
-const HealthCard: React.FC<HealthCardProps> = ({ watchAlerts, stopWatchAlerts, alertsResults }) => {
+const HealthCard: React.FC<HealthCardProps> = ({
+  watchAlerts,
```
Comment (dtaylor113) [inline frontend/packages/metal3-plugin/src/components/baremetal-hosts/dashboard/StatusCard.tsx]: Where is `watchAlerts` defined?  It seems to have been removed from `with-dashboard-resources.tsx`?

File: frontend/public/actions/dashboards.ts (hunk 1 above)
Comment (rawagner) [reply frontend/public/actions/dashboards.ts]: good catch! Removed

**Pull request merged**